scala/scala-impl/src/org/jetbrains/plugins/scala/lang/formatting/scalafmt/dynamic/ScalafmtDynamicDownloader.scala

```diff
@@ -54,8 +54,18 @@ object ScalafmtDynamicDownloader {
   ) extends DependencyManagerBase {
 
     // first search in the provided resolvers, then fallback to the default ones
+    // but only use resolvers with http(s) protocol, to avoid problems with private repositories.
+    // (like gcs, s3, etc.) This is fine since scalafmt should always be published to a public repo.
```

Comment:
This showstopper bug has made our life with IntelliJ harder for a long time. Thanks for the fix proposal! 
LGTM!

A side note. Probably not a big deal here however, the pattern matching allows us to simplify it as follows:
```scala
    val validUrl = "http[s]{0,1}://.*".r

    override protected def resolvers: Seq[Resolver] =
      (extraResolvers ++ super.resolvers).filter {
        case MavenResolver(_, validUrl(_*)) => true
        case IvyResolver(_, validUrl(_*)) => true
        case _ => false
      }
```

Reply:
I slightly changed the style, but I like the idea. 👍

Reply:
All right, it looks good 👍